process/project-lifecycle.md

```diff
@@ -0,0 +1,178 @@
+* Projects must be able to show a consistent release cadence.
+* Projects must have documented project governance and be able to demonstrate that governance in action.
+* When applicable, projects must have completed a security audit through a third party and addressed audit findings and recommendations
+
+#### Project Graduation Process: Incubating to Graduation
+
+Graduation requires undergoing due diligence as a part of the process to move from Incubation to Graduation. Due Diligence is driven by a TAC or parent WG sponsor. For projects seeking Graduation, this may be a light refresh of the existing Due Diligence to cover the additional criteria, or a more in depth Due Diligence depending on the level of change the project has incurred since the original Due Diligence of Incubation was performed. Once the diligence is confirmed by the Sponsor to be complete and the proposal made, the Due Diligence document is made available to the community for two weeks the solicit public comment before a TAC vote is called.
+
+
+
+### Archived
+
+Open source projects have a lifecycle and there are times that projects become inactive due to a variety of reasons. There are also cases where a project may no longer want to be supported by the OpenSSF, or the OpenSSF TAC may no longer wish to recommend the use of a project. Archiving happens through a vote of the TAC, and can be requested by the corresponding project's lead(s) or a TAC member.
+
+#### Archiving Considerations
+
+When voting on a proposal to archive a project, TAC members may consider:
+* If the project has consistently met the Project Requirements
+* If the project still supports the OpenSSF mission
+* If the project's inactivity or inconsistent maintenance presents a user security risk
+* If the project's design or approach is no longer a recommended best practice
+
+#### Archive Process
+
+Archived projects are no longer in active development and are only archived after a TAC vote.
+
+What does archiving for an OpenSSF project mean?
+* OpenSSF will no longer provide budgetary, event, or marketing support for the project
+* Trademarks and domain names of archived projects are still hosted by the OpenSSF and the Linux Foundation
+* Project maintainers may request trademark transfers from the OpenSSF and the Linux Foundation
+* OpenSSF can provide services such as documentation updates to help transition users
+* Other OpenSSF marketing activities will no longer be provided for the project
+
+To archive a project:
+* A proposal must be put forth to the TAC or WG repo
+* The TAC or parent WG will inform the project maintainers, OpenSSF end user community and wider community of all archiving proposals
+* The proposal must remain open for at least 2 weeks of discussion after the maintainers are informed.
+* A vote must be finalized with 2/3 approval from the TAC or parent WG.
```

Comment:
Nit: Previous two items don't have periods, these two do.

Reply:
agreed.  #dislike mismatched punctuation style.  probably should go back through all lists and ensure they are all done the same.

Reply:
Addressed in PR#117